index.js

```diff
@@ -1,88 +1,119 @@
 // ***** Question 1 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 1", "color: red") 
+var counter = 1
+console.log("%cQuestion 1", "color: red") 
 
-// console.log(counter) 
-// // => 2
-// console.log("%c----------", "color: red") 
+counter = 2
+console.log(counter) 
+// => 2
+console.log("%c----------", "color: red") 
 
 
 // ***** Question 2 *****
 // *** Uncomment the lines below to test
 // *** after testing, comment the line causing the error back in 
 // *** otherwise, the error will stop the rest of your code from running
-// console.log("%cQuestion 2", "color: red") 
+console.log("%cQuestion 2", "color: red") 
+const name = "Jahaira"
+console.log(name)
 
-// console.log(name)
-// => "Raffy"
 // name = "Not Raffy" 
 // => TypeError
-// console.log("%c----------", "color: red") 
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 3 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 3", "color: red") 
+console.log("%cQuestion 3", "color: red") 
 
-// console.log(drinkWater(12))
-// // => "Man I sure am thirsty"
-// // => "Ahh that hits the spot"
-// // => 11
-// console.log("%c----------", "color: red") 
+// let currentThirstLevel = 5; 
+
+function drinkWater(currentThirstLevel) {
+  if (currentThirstLevel < 15) {
+    return "Man I sure an thirsty"}
+    else {
+    return " Ahh that hits the spot"
+  }
+}
+console.log(drinkWater(15))
+// => "Man I sure am thirsty"
+// => "Ahh that hits the spot"
+// => 11
+console.log("%c----------", "color: red") 
 
 
 
 // ***** Question 4 *****
 // *** Uncomment the lines below to test
-// console.log("%cQuestion 4", "color: red") 
-
-// console.log(sameSameButDifferent(5, 5)) 
-// // => "same same"
+console.log("%cQuestion 4", "color: red") 
+
+function sameSameButDifferent(num, maybeNum) {
+  if (num === maybeNum) {
+    return  "same same";
+  } else if (num == maybeNum) {
+    return "same same (but different)";
+  } else { return "different"}
+}
+console.log(sameSameButDifferent(5, 5)) 
```

Comment:
This is perfect. The key was to demonstrate the difference between == and ===. Now that you have done that, try refactoring the else if using parseInt.

To go deeper, use typeof in the conditional to determine the data type
e.g.
if (typeof num === 'number' && typeof maybeNum === 'number' && num === maybeNum)